README.md

```diff
@@ -1,2 +1,457 @@
+
+# Installation
+
+To install the SDK, you will need to be using [Composer](http://getcomposer.org/)
+in your project.
+If you aren't using Composer yet, it's really simple! Here's how to install
+composer:
+
+```bash
+curl -sS https://getcomposer.org/installer | php
+```
+
+Use Composer to install the package:
+
+```bash
+composer require unione/unione-php
+```
+
+## Usage
+
+### Here's how to send a message using the SDK:
+More about the **UniOne** send mail API [https://docs.unione.io](https://docs.unione.io/en/web-api-ref#email).
+
+```php
+ $recipients = [
+    ["email" => "example@example.com"]
+  ];
+
+  $body = [
+    "html" => "<b>Test mail, {{to_name}}</b>",
+    "plaintext" => "Hello, {{to_name}}",
+    "amp" => "<!doctype html><html amp4email><head> <meta charset=\"utf-8\"><script async src=\"https://cdn.ampproject.org/v0.js\"></script> <style amp4email-boilerplate>body[visibility:hidden]</style></head><body> Hello, AMP4EMAIL world.</body></html>"
+  ];
+
+  $mail = new Email($recipients, $body);
+  $mail->setFromEmail('test@example.com');
+
+  // https://us1.unione.io/en/transactional/api/v1/ - UniOne USA & Canada Instance
+  // https://eu1.unione.io/en/transactional/api/v1/ - UniOne European Instance
+  // First, instantiate the SDK with your API credentials
+  $client = new UniOneClient('https://eu1.unione.io/en/transactional/api/v1/', 'api-key');
+  $mailer = $client->emails();
+
+  // Now, compose and send your message.
+  $response = $mailer->send($mail);
+```
+
+### Here's how to send a subscribe message using the SDK:
+More about the **UniOne** send mail API [https://docs.unione.io](https://docs.unione.io/en/web-api-ref#email-subscribe).
+
+```php
+  $requestBody = [
+  "from_email" => "test@example.com",
+  "from_name" => "string",
+  "to_email" => "test@example.com"
+  ];
+
+  $client = new UniOneClient('https://eu1.unione.io/en/transactional/api/v1/', 'api-key');
+  $mailer = $client->emails();
+
+  // Now, send your subscribe message.
+  $response = $mailer->subscribe($requestBody);
+```
+
+### Here's how to set a template using the SDK:
+More about the **UniOne** Template API [https://docs.unione.io](https://docs.unione.io/en/web-api-ref#template).
+```php
+  $requestBody = [
+      "template" => [
+        "name" => "First template",
+        "body" => [
+          "html" => "<b>Hello, {{to_name}}</b>",
+          "plaintext" => "Hello, {{to_name}}",
+          "amp" => "<!doctype html><html amp4email><head> <meta charset=\"utf-8\"><script async src=\"https://cdn.ampproject.org/v0.js\"></script> <style amp4email-boilerplate>body[visibility:hidden]</style></head><body> Hello, AMP4EMAIL world.</body></html>"
+        ],
+        "subject" => "Test template mail",
+        "from_email" => "test@unione.devbranch.work",
+        "from_name" => "Devbranch",
+      ]
+    ];
+
+  $client = new UniOneClient('https://eu1.unione.io/en/transactional/api/v1/', 'api-key');
```

Comment:
Let's do not repeat the same code over and over again. We can assume the client is already instantiated.